spark/.gitignore

```diff
@@ -0,0 +1,41 @@
+# use glob syntax.
+syntax: glob
+*.ser
+*.class
+*~
+*.bak
+#*.off
+*.old
+
+# eclipse conf file
```

Comment:
Please could you update the main .gitignore file for the repository instead of adding an extra spark/.gitignore file and then make sure your .settings, .project, .classpath and other local files are removed.